src/spicy/.clang-format

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
Ok, I like this. Then I'll hold off on further coding convention things altogether for now, so that we can more easily port code back and forth for a little while.

>  if you could somewhere in a commit message mention the exact zeek/spicy-plugin commit you took the sources from

It's in there, it's `d8c296b81cc2a11`.